### DISCUSSION TITLE
Github Copilot suddenly stops giving suggestions in Visual Studio

### DISCUSSION THREAD

**Discussion by NorskIT** [community/community#45052] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

As title says, the copilot suddenly stops giving suggestions. A restart of VS fixes the issue, but Id like to not restart the whole IDE when it stops working. Is there any hotkeys to "restart" the copilot, or a command?

**Comment by jge162** [comment, marked as answer]
@NorskIT you can go to the "Edit" menu and select "IntelliSense" and then "Restart."

You can also try using the keyboard shortcut "Ctrl+Shift+R" to restart IntelliSense.

Another option is to disable and re-enable the IntelliSense feature by going to "Tools" > "Options" > "Text Editor" > "C#" > "IntelliSense" and then unchecking and rechecking the "Enable IntelliSense" checkbox.

**Reply by NorskIT** [reply]
Hi @jge162 , thanks for reply.
Will restarting the IntelliSense affect the Github Copilot? I will try this out.

**Reply by jge162** [reply]
Hi @NorskIT Restarting the IntelliSense feature in Visual Studio should not affect the Github Copilot. IntelliSense is a feature built into Visual Studio that provides code suggestions, and the Github Copilot is an extension that runs on top of Visual Studio and provides additional functionality such as code review suggestions and pull request templates.

**Reply by jge162** [reply]
@NorskIT However, it's possible that the Github Copilot relies on the IntelliSense feature to provide some of its functionality, and restarting IntelliSense may help resolve any issues with the Github Copilot not providing suggestions.

**Reply by NorskIT** [reply]
It fixed the problem, thanks @jge162.

**Reply by jge162** [reply]
Nice. Happy coding!
